core-web/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+.cursor/rules/nx-rules.mdc
+.github/instructions/nx.instructions.md
```

Comment:
Why? we should push this.

Reply:
oh this was nx, i have to remove this change